src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -206,6 +206,13 @@ private BoundExpression CheckValue(BoundExpression expr, BindValueKind valueKind
                         }
                     }
                     break;
+
+                case BoundKind.UnconvertedObjectCreationExpression:
+                    if (valueKind == BindValueKind.RValue)
+                    {
+                        return expr;
+                    }
+                    break;
```

Comment:
can this go in separately?

Reply:
This change fixes the semantic model which affects the completion scenario for parameter names `new(Alice: ...`